src/index.html

```diff
@@ -1,12 +1,54 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="calendar__month calendar__month--month-length--28">
```

Comment:
```suggestion
      <div class="calendar calendar--month-length--28 calendar--start-day--thu">
```
Consider changing class names to match the task description